.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "dlib"]
+	path = dlib
+	url = https://github.com/davisking/dlib.git
+[submodule "freetype2"]
+	path = freetype2
+	url = https://github.com/glaba13/freetype2.git
```

Comment:
Please transfer these repositories to stream-labs if possible.

Reply:
Don't have a permission to create a stream-labs repository
If access is given or created repositories I can change links